Frends.Csv/Definitions.cs

```diff
@@ -158,6 +157,12 @@ public class ParseOption
         /// </summary>
         public string CultureInfo { get; set; } = "";
 
+        /// <summary>
+        /// A flag to let the reader know if quotes should be ignored.
+        /// </summary>
+        [DefaultValue("false")]
+        public bool IgnoreQuotes { get; set; } = false;
```

Comment:
Please add examples to each parameters.

Reply:
Legacy documentation doesn't use examples.